pkg/rancher-desktop/main/commandServer/httpCommandServer.ts

```diff
@@ -18,6 +19,11 @@ import type { Settings } from '@pkg/config/settings';
 import type { TransientSettings } from '@pkg/config/transientSettings';
 import type { DiagnosticsResultCollection } from '@pkg/main/diagnostics/diagnostics';
 
+export type BackendState = {
+  vmState: State,
+  locked: boolean,
```

Comment:
I'm fine with the choice of `locked` here, because it's clearly scoped by the type.

Reply:
Given the other changes, should this turn into `lockedReason: string`? (Totally optional; a bool is good.)

Reply:
For this PR, let's leave it as `locked`, since it is just a boolean. In subsequent work, it sounds like we will be changing this field so that it is a string, with an empty string indicating an unlocked state. At that time it will definitely make sense to change its name to `lockedReason`.